webpack.config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  entry: {
+    rollbar: './src/bundles/rollbar.js',
+    'rollbar.nojson': './src/bundles/rollbar.nojson.js',
+    'rollbar.umd': './src/bundles/rollbar.umd.js'
+  },
+  output: {
+    library: 'Rollbar',
+    path: './dist/bundles/',
```

Comment:
@coryvirok yah, I'll change this. This was only to don't have conflicts with the existing ones while the development process.